guides/slido-moderation.rst

```diff
@@ -0,0 +1,102 @@
+Sli.do moderation guide for PyCon CZ
+====================================
+
+
+The main reason for question moderation is that some people like to hide behind
+the anonymity and be rude. The second, less important reason is they ask
+questions that are unrelated to the topic. Here's a basic guide followed by some
+examples of questions and comments that should and should not be approved (some
+of them we actually got as questions, some are made up):
+
+
+How to ask / vote on questions
+------------------------------
+
+1) Go to `www.sli.do` and use ``#pyconcz`` code to join the event
+2) Choose the room you are in.
+3) Type your question or upvote already existing ones.
+
+
+How to setup presentation wall view (used on screen after speaker's slides end)
+-------------------------------------------------------------------------------
+
+1) Follow steps 1 and 2 from ``How to ask`` guide.
+2) Click on the tiny link at the bottom left side that says ``Present view``.
+3) Switch the room (right top corner) if needed (the wall doesn't care about the
+   choice you made when entering the event).
+
+
+How to moderate
+---------------
+
+1) Go to `www.sli.do` and log in (ask for credentials if you don't have them,
+   @veruu is your main contacts but every moderator should have them).
```

Comment:
```suggestion
   @veruu is your main contact but every moderator should have them).
```

Reply:
Yay someone is actually reading the text. I removed people from the outdated contact list and forgot to change part of the wording, thanks for noticing